internal/clients/nightfall/nightfall.go

```diff
@@ -192,6 +196,26 @@ func (n *Client) createScanRequest(items []string) nightfallAPI.ScanRequest {
 	}
 }
 
+func (n *Client) scanContent(ctx context.Context, cts []*contentToScan, requestNum int, logger logger.Logger) ([]*diffreviewer.Comment, error) {
+	// Pull out content strings for request
+	items := make([]string, len(cts))
+	for i, item := range cts {
+		items[i] = item.Content
+	}
+
+	// send API request
+	resp, err := n.Scan(ctx, logger, items)
+	if err != nil {
+		logger.Debug(fmt.Sprintf("Error sending request number %d with %d items", requestNum, len(items)))
```

Comment:
hmm not sure I agree with this being a debug log - (to me) it seems like it would be a good thing to always surface this error to the customer if we encounter it, so I think we either want to explicitly log an Error here, or return and let the parent log an Error.

What do you think?

Reply:
Yea, the idea for this debug is that it would give more insight to where exactly the call failed and since the error is returned it is handled appropriately by the parent caller.